Userland/Libraries/LibLine/Editor.h

```diff
@@ -420,6 +421,7 @@ class Editor : public Core::EventReceiver {
     Vector<char, 512> m_incomplete_data;
     Optional<Error> m_input_error;
     ByteString m_returned_line;
+    ByteString m_last_erased;
```

Comment:
Please store this as a `Vector<u32>` instead, the conversion from utf32 <-> utf8 is not necessary as this is entirely internal.

Reply:
Did it. Used `clear()`. Or should i use `clear_with_capacity()`?

Reply:
I guess `clear_with_capacity` will be a little nicer on the allocator.